spec/src/main/asciidoc/configuration.asciidoc

```diff
@@ -104,3 +104,7 @@ public class MyClient{
 
 }
 ----
+
+If an annotation is not present, the configured properties are ignored. 
+For an instance, the property `com.acme.ClassA/methodB/Retry/maxRetries` will be ignored if `Retry` annotation is not specified on the `methodB` of `com.acme.ClassA`. 
```

Comment:
I'm not an english native speaker, but I find that "for an instance" sounds strange compared with "for instance", we should check that

> ignored` if `@Retry`...

IMO we should add the `@` before annotations name to make the doc easier to read.

Reply:
Correct, it should be "for instance, if the property `com.acme.ClassA/methodB/Retry/maxRetries` is set it will be ignored if the `Retry` annotation is not specified on either the method `methodB` or the class `com.acme.ClassA`

Reply:
great. thanks guys:o!